### DISCUSSION TITLE
Show Full Day Events at the bottom of the day

### DISCUSSION THREAD

**Discussion by sparkyparkes** [MMRIZE/MMM-CalendarExt3#196] [Q&A]
Looking at the month view, I'm trying to figure out a way to show the full day events at the bottom of the day rather than the top.  Is this possible?

**Comment by eouia** [comment]
Sorry for the late reply. I was moving the country from Germany to S.Korea so I had no time to care about this project. 
Generally, it's not possible because this module would stack the event block as a masonry method by its longevity to fill the empty space, so if the light(short-term) event stacks first (at the top), the whole algorithm would crash, and there could be some waste of space.

**Comment by sparkyparkes** [comment]
That makes sense. Thanks for reply!
